eth/backend.go

```diff
@@ -234,6 +237,17 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 	legacyPool := legacypool.New(config.TxPool, eth.blockchain)
 
 	eth.txPool, err = txpool.New(config.TxPool.PriceLimit, eth.blockchain, []txpool.SubPool{legacyPool, blobPool})
+
+	if !config.TxPool.NoLocals {
```

Comment:
Is it possible to move the local tracker into the `txpool.Pool`?

I think this is an internal mechanism of the transaction pool and would be better “hidden” under the hood.